Git-Math/qautomata
# Run the application on a web page

The application is now running on a web browser using web assembly.

To test the web ui:
- install `npm` and `node` > 18
- move to the ui folder `cd ./ui`
- install deps `npm install`
- run the web app `npm run start`

To test the application:
- from the root of the repository run `cargo run -p ui`

Note that this won't work if you are in the `ui` folder, this will be addressed on a subsequent PR.


We drew inspiration from the following projects:
- https://github.com/JulianCataldo/astro-nannou-starter
- https://github.com/tomoyanonymous/nannou-web-template
- https://github.com/steezeburger/siqnastee-nannou-web
